PlatformSpecific/WindowsGameWindowManager.cs

```diff
@@ -121,12 +129,15 @@ public void ShowWindow()
         /// <summary>
         /// Flashes the game window on the taskbar.
         /// </summary>
+#if !NETFRAMEWORK
+        [System.Runtime.Versioning.SupportedOSPlatform("windows5.1.2600")]
+#endif
         public void FlashWindow()
         {
             if (gameForm == null)
                 return;
 
-            WindowFlasher.FlashWindowEx(gameForm);
+            _ = WindowFlasher.FlashWindowEx(gameForm.Handle);
```

Comment:
I'm aware the latest MS C# style guide says otherwise, but I'm not a fan of these discards.